src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -1040,15 +1044,19 @@ private IContent ValidateMoveOrCopy(MoveOrCopy model)
             return toMove;
         }
 
-        private void ShowMessageForPublishStatus(PublishStatus status, INotificationModel display)
+        private void ShowMessageForPublishStatus(PublishStatus status, INotificationModel display, DateTime? expireDate)
         {
             switch (status.StatusType)
             {
                 case PublishStatusType.Success:
-                case PublishStatusType.SuccessAlreadyPublished:
+                case PublishStatusType.SuccessAlreadyPublished:                    
                     display.AddSuccessNotification(
                             Services.TextService.Localize("speechBubbles/editContentPublishedHeader"),
-                            Services.TextService.Localize("speechBubbles/editContentPublishedText"));
+                            expireDate.HasValue
+                                ? string.Format(Services.TextService.Localize("speechBubbles/editContentPublishedWithExpireDateText"),

```

Comment:
@kjac This is not the correct way to localize text with tokens you'll need to fix this up. You'll see that tokens within the text files are defined like `%0%` or `%1%` (based on the parameter index), then you will pass in an object of parameters to the `Localize` method on `TextService`

Let me know if you have other questions for this, thanks!

Reply:
@Shazwazza my bad. I'll fix in another PR. Thanks for reviewing 👍

Reply:
PR in #3840